ConsumerApi.Sdk/Endpoints/Files/Types/CreateFileRequest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Backbone.ConsumerApi.Sdk.Endpoints.Files.Types;
+
+public class CreateFileRequest
+{
+    public required FileData Content { get; set; }
```

Comment:
I would only accept a stream here. The `FileData` class barely brings any advantage. It just enables you to write

```c#
new CreateFileRequest 
{
    Content = new FileData("/path/to/file")
}
```

instead of 

```c#
new CreateFileRequest 
{
    Content = File.OpenRead("/path/to/file")
}
```

Also, there could be some other places you could read file content from (e.g. the assembly, the network, ...). And we surely don't want to create a constructor for all of them.

Reply:
👍